tests/bin/run-e2e-travis.sh

```diff
@@ -1,12 +1,7 @@
 #!/usr/bin/env bash
-if [[ ${RUN_E2E} == 1 ]]; then
```

Comment:
@budzanowski A little background story on this script and the `RUN_E2E` check.

We added it here because we wanted it to only run on 1 of the matrices, namely, the latest PHP, https://github.com/Automattic/woocommerce-services/blob/93d2d9e7f0699275239b7a921af1b24a6d0f2054/.travis.yml#L53. 

We might want to check with @juliaamosova if we want to run on all versions.

Reply:
I may have done something awkward sometime ago. For sure we want to run on only one.